src/library/Uno.Themes.WinUI.Markup/Theme/TextBox.cs

```diff
@@ -0,0 +1,207 @@
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxDeleteButtonForegroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxDeleteButtonForegroundDisabled", true);
+					}
+
+					public static class Background
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackground")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxBackground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxBackgroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxBackgroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBackgroundDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxBackgroundDisabled", true);
+
+					}
+
+					public static class BorderBrush
+					{
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrush")]
+						public static ResourceValue<Brush> Default => new("FilledTextBoxBorderBrush", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushPointerOver")]
+						public static ResourceValue<Brush> PointerOver => new("FilledTextBoxBorderBrushPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushFocused")]
+						public static ResourceValue<Brush> Focused => new("FilledTextBoxBorderBrushFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "FilledTextBoxBorderBrushDisabled")]
+						public static ResourceValue<Brush> Disabled => new("FilledTextBoxBorderBrushDisabled", true);
+					}
+
+					public static class Typo
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "FilledTextBoxFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("FilledTextBoxFontFamily", true);
+
+						[ResourceKeyDefinition(typeof(FontWeight), "FilledTextBoxFontWeight")]
+						public static ResourceValue<FontWeight> FontWeight => new("FilledTextBoxFontWeight", true);
+
+						[ResourceKeyDefinition(typeof(double), "FilledTextBoxFontSize")]
+						public static ResourceValue<double> FontSize => new("FilledTextBoxFontSize", true);
+
+						[ResourceKeyDefinition(typeof(double), "FilledTextBoxCharacterSpacing")]
+						public static ResourceValue<double> CharacterSpacing => new("FilledTextBoxCharacterSpacing", true);
+
+						[ResourceKeyDefinition(typeof(Thickness), "FilledTextBoxBorderThicknessNormal")]
+						public static ResourceValue<Thickness> Normal => new("FilledTextBoxBorderThicknessNormal", true);
+
+						[ResourceKeyDefinition(typeof(Thickness), "FilledTextBoxBorderThicknessFocused")]
+						public static ResourceValue<Thickness> Focused => new("FilledTextBoxBorderThicknessFocused", true);
```

Comment:
```
<StaticResource x:Key="FilledTextBoxBorderThicknessNormal" ResourceKey="TextBoxOutlinedStrokeHeight" />
<StaticResource x:Key="FilledTextBoxBorderThicknessFocused" ResourceKey="TextBoxFocusStrokeWidth" />
```

Should be outside of the `Typo` class as it is related to `Border` and not `Typography`

Reply:
fixed